recipe/meta.yaml

```diff
@@ -30,7 +30,13 @@ requirements:
     - trio >=0.17,<1.0
     - trio-websocket >=0.9,<1.0
     - urllib3 >=1.26,<2.0
-
+    # urllib3[secure]
```

Comment:
Hmm... looks like upstream has these by default?

https://github.com/conda-forge/urllib3-feedstock/blob/main/recipe/meta.yaml#L23-L32

Reply:
Yeah but, if they ever remove it, it belongs in here if we are to "respect" the PyPI way of handling metadata :-/

Reply:
If you agree can you merge this one?

Reply:
maybe we get that to be split up into separate outputs, e.g. `urrlib3-with-socks` so we're not duplicating the constraints.

Reply:
That is a long standing issue with conda/conda-forge. I'd love for a better support but yeah, maybe multiple outputs is the best we'll ever have.

Reply:
This discussion is actually timely, we're deprecating the `[secure]` extra as it's no more secure for almost all Python interpreters in use and requires time to maintain that's better spent elsewhere: https://github.com/urllib3/urllib3/issues/2680

We've got a separate issue to help migrate packages using `urllib3[secure]` (like Selenium!) away from the dependency: https://github.com/urllib3/urllib3/issues/2700